src/components/actions/filters/ItemFilter.tsx

```diff
@@ -13,21 +15,27 @@ export const ELLIPSIS: string = '...';
 export class ItemFilter extends React.Component<IItemFilterProps, any> {
 
   render() {
-    let itemFilter: string = this.props.item;
+    let itemFilterText: string = this.props.item;
     if (this.props.crop) {
-      let itemFilterLength: number = itemFilter.length;
+      let itemFilterLength: number = itemFilterText.length;
 
       if (itemFilterLength > Math.abs(this.props.crop)) {
-        itemFilter = this.props.crop > 0
-          ? itemFilter.substring(0, this.props.crop) + ELLIPSIS
-          : ELLIPSIS + itemFilter.substring(itemFilterLength + this.props.crop, itemFilterLength);
+        itemFilterText = this.props.crop > 0
+          ? itemFilterText.substring(0, this.props.crop) + ELLIPSIS
+          : ELLIPSIS + itemFilterText.substring(itemFilterLength + this.props.crop, itemFilterLength);
       }
     }
 
+    const itemFilter: JSX.Element = this.props.itemTooltipProps
+      ? <Tooltip {...this.props.itemTooltipProps}>
```

Comment:
if you itemTooltipProps title is empty or undefined. The child element will be wrap by a span tag. I dont think this condition is necessary. You can test but you probably can use <Tooltip/> in both case